react/week3/todo-app/src/components/AddTodo.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from "react";
+
+const AddTodo = ({ addTodo }) => {
+    const [description, setDescription] = useState("");
+    const [deadline, setDeadline] = useState("");
+    const [visible, setVisible] = useState(false)
+
+    const minDate = () => {
+        const day = new Date();
+        let dd = day.getDate();
+        let mm = day.getMonth() + 1; //January is 0!
+        const yyyy = day.getFullYear();
```

Comment:
Clear and descriptive naming with these variables - and happy to see the comment!